src/app/layout/header/header.component.html

```diff
@@ -199,15 +197,17 @@ <h4 class="mb-0 f-16 color-white">Notification</h4>
     <div class="border-bottom-1">
         <div class="d-flex flex-column align-items-center m-3">
             <img class="img-thumbnail rounded-circle mt-2 mb-3" src="../assets/img/identity-image.png" width="44"
-                height="44" alt="avatar">
-            <div class="label-role" *ngIf="!userName">Manage Profile</div>
-            <div class="label-role" *ngIf="userName"> {{ userName }} </div>
-        </div>
-        <div class="d-flex align-items-center opacity-50 text-truncate-middle did-wrapper mx-3 mb-3"
-            matTooltip="Copy to clipboard" [cdkCopyToClipboard]="userDid" (cdkCopyToClipboardCopied)="didCopied()">
-            <div class="f-14">{{userDid | didFormatMinifier}}</div>
-            <mat-icon class="ml-2">content_copy</mat-icon>
+                 height="44" alt="avatar">
+            <div class="label-role">{{ userName$ | async}}</div>
```

Comment:
duplicate 'async' (185/201), it will be better to combine 'async'

Reply:
You want to make one async like this: `<ng-container *ngIf="userName$ | async as userName"> everything from component </ng-container>` ?